acceptance_tests/utilities/jwe_helper.py

```diff
@@ -16,3 +17,17 @@ def decrypt_signed_jwe(signed_jwe: str) -> Mapping:
 
     # Extract, deserialize, and return the payload
     return json.loads(jws_token.payload)
+
+
+def decrypting_token_and_asserts(context, query_strings: dict):
```

Comment:
In general, we avoid passing the context into helper functions like this since it obscures the signature and side effects of the helpers